graylog2-plugin-interfaces/src/main/java/org/graylog2/plugin/Message.java

```diff
@@ -65,7 +66,7 @@
     public static final String FIELD_LEVEL = "level";
     public static final String FIELD_STREAMS = "streams";
 
-    private static final Pattern VALID_KEY_CHARS = Pattern.compile("^[\\w\\.\\-@]*$");
+    private static final Pattern VALID_KEY_CHARS = Pattern.compile("^[\\w\\.\\-@/]*$");
```

Comment:
I think we should just remove the `/` right know and not allow more characters at the moment. Because we do not know yet how other characters might behave when searching, filtering etc.

I also still see this as a temporary solution that might not be in the final 2.0 because there are way to many ways to still introduce `.` character in field names. In the end this needs to be done when adding fields to the message object.
Until we are there, this approach is okay in my opinion.